Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -992,6 +993,12 @@ extension TelemetryWrapper {
             GleanMetrics.FirefoxHomePage.historyHighlightsGroupOpen.record()
         case (.action, .view, .historyImpressions, _, _):
             GleanMetrics.FirefoxHomePage.customizeHomepageButton.add()
+        case (.action, .view, .historyHighlightContextualMenu, _, let extras):
+            if let type = extras?[EventExtraKey.contextualMenuType.rawValue] as? String {
+                GleanMetrics.FirefoxHomePage.historyHighlightsContext.record(GleanMetrics.FirefoxHomePage.HistoryHighlightsContextExtra(type: type))
```

Comment:
nit: For readability we could add a separate line to create the extra
let contextExtra = GleanMetrics.FirefoxHomePage.HistoryHighlightsContextExtra(type: type)